nubilanding/src/components/recargas/recargas.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styles from '../recargas/styles.module.scss';
+import LogoPlay from '../../assets/img_logo_playstation.png';
+import LogoForever from '../../assets/img_logo_forever21.png';
+import LogoTwitch from '../../assets/img_logo_twitch.png';
+import LogoTripAdvisor from '../../assets/img_logo_tripadvisor.png';
+import LogoAirbnb from '../../assets/img_logo_airbnb.png';
+import LogoEbay from '../../assets/img_logo_ebay.png';
+import LogoTiendaMia from '../../assets/img_logo_tiendamia.png';
+import LogoCarters from '../../assets/img_logo_carters.png';
+import LogoOrigin from '../../assets/img_logo_origin.png';
+
+class Recargas extends React.Component {
+    render () {
+        return (
+            <div className={styles.contenedorGeneral}> 
+                <div className={styles.contenedorRecarga}>
+                    <h2 className={styles.titulos}>Recargá tu cuenta PayPal</h2>
+                    <p className= {styles.textoRecargas}>
+                        Con Nubi agregá dinero a tu cuenta PayPal desde tu banco y comprá en todo el mundo sin tajeta de crédito. ¡Si no tenés cuenta PayPal creátela desde Nubi en unos simples pasos! 
+                    </p>
+                    <p className= {styles.textoRecargas}>
+                        El costo de la recarga es de 5% + IVA sobre la tarifa.
+                    </p>
+                    <button className={styles.boton}>Registrate Gratis</button>
+                </div>
+                <div className={styles.contenedorAhorra}>
+                    <h2 className={styles.titulos}>Ahorrá en tus compras online</h2>
+                    <div className={styles.contenedorLogosPF}>
+                        <img className={styles.logos} src= {LogoPlay} alt="Logo Playstation Store"/>
+                        <img className={styles.logos} src= {LogoForever} alt="Logo Forever 21"/>
+                        <img className={styles.logos} src= {LogoTwitch} alt="Logo Twitch"/>
+                        <img className={styles.logosF} src= {LogoTripAdvisor} alt="Logo Trip Advisor"/>
+                    </div>
+
+                    <div className={styles.contenedorLogosSF}>
```

Comment:
Acá también podrías hacer uso de los [pseudo selectores](https://www.w3schools.com/cssref/sel_last-child.asp) para seleccionar el segundo elemento con la misma clase. Así no tienes que crear ninguna clase adicional y es más fácil entender el código desde el CSS.

Reply:
Para lo contenedores (tanto para el que es actualmente `contenedorLogosPF` y `contenedorLogosSF`) puedes usar la misma clase de contenedor:
```suggestion
                    <div className={styles.contenedorLogos}>
```

Luego en el SCSS seleccionarías el último usando nuevamente el `:last-child`:
```
.contenedorLogos:last-child { ... }
```